src/overview/actions.js

```diff
@@ -262,7 +266,14 @@ export const showTags = index => (dispatch, getState) => {
         dispatch(setActiveTagIndex(index))
     }
 }
-
+export const openShare = index => (dispatch, getState) => {
```

Comment:
You can insert one empty line before and after the function.

Reply:
Done